### PR TITLE
F 9

### PR DESCRIPTION
**Pull request by graywolf**
@voloda can you take a look?

### CODE REVIEW

File: 08-Log/readme.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	diff --git a/MyWindowsApp/code.txt b/MyWindowsApp/code.txt
+	new file mode 100644
+	index 0000000..e69de29
+
+> I'm not using `--oneline` here, since without colors it gets kinda dense.
+
+More informations
+-----------------
+
+I recommend checking [git-log documentation](https://git-scm.com/docs/git-log) for more details, there is a **lot** of options for `git log`.
```
Comment (voloda) [inline 08-Log/readme.md:114]: Perhaps adding `git log --name-status` would be useful as well

File: 09-Blame/readme.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+git-blame
+=========
+
+* Show what revision and author last modified each line of a file
+
+This is one of the more useful commands in git arsenal. It gives you information on who, when and in what revision did last modification to that specific line. If follows whole-file renames and can even track lines moved/copied from other files (you know, for those situations when you have one file copied into 4 with just minor modifications).
```
Comment (voloda) [inline 09-Blame/readme.md:8]: Should you mention that it is the same function as TFS annotate?

File: 09-Blame/readme.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+What we can see here?
+
+1. in what commit the change was made
+2. who made it
+3. when
+4. line number
+5. line content
+
+This seems useful, right? But wait, we can do more! We all know that it shouldn't be done but it's still is. Copy&paste programming is sad reality and can really complicate finding out who originaly wrote what line and where. But git to the rescue! First let's run blame on `MyWindowsApp/dll.txt` to see the result:
```
Comment (voloda) [inline 09-Blame/readme.md]: > it's still is

should be: it still is ?

**Comment by voloda** [comment]
Overall this looks good with respect to content. Please review above comments.

I am only wondering that we should use 'common' style/formatting. I plan to do some final review/clenaup once we are done but we should go in the same direction.

So which of two styles would you pick and why? :)

File: 09-Blame/readme.md (hunk 3 above)
Comment (graywolf) [inline 09-Blame/readme.md]: english hard

**Comment by graywolf** [comment]
I made changes to satisfy your feedback, take a look at it.

Well, since this was supposed to be useful also for self-study, I think my style with examples of how the result looks is more useful, since person can verify what he did mostly works.

**Comment by voloda** [comment]
Look good.

Regarding style/formatting.

I like the sample output and I will update that. Though - it has two more parts:
- First I am using bullet points while you are telling a story
- Second I was trying to put together 'scenarios' while yours are rather showing commands
  - It is kind of natural with blame/log but more complex with others

Overall all 'lessons' should looks similar to make them easy to read. Let's continue and we will see.

**Comment by graywolf** [comment]
yep, I agree that for more complex stuff (merge, rebase, cherry-pick, ..) actual scenario using some sample c++ code would be better

**Pull request merged**